### ISSUE TITLE
Unable to save submissions with images in control panel

### ISSUE THREAD

**Issue by d-karstens** [verbb/formie#1054]
### Describe the bug

Hi, after updating to Formie 1.6.7 we are unable to create or edit a submission with an image in the control panel. Though submitting on the front end still works

The only thing I can see in the logs is
`2022-08-24 11:52:22 [-][1][-][error][formie] Couldn’t save submission - [].`

Downgrading to 1.6.6 seems to fix the issue

Let me know if you need any more info

### Steps to reproduce

1. Try to create or edit a submission with an image in control panel
2. See error

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

3.7.52

### Plugin version

1.6.9

### Multi-site?

No

**Comment by Ocupe** [comment]
We see exactly the same problem.

**Comment by engram-design** [comment]
This was due to an added check to protect against abusing the File Upload field. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by d-karstens** [comment]
Cheers for the quick fix!

**Comment by engram-design** [comment]
Fixed in [1.6.10](https://github.com/verbb/formie/releases/tag/1.6.10)
